src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -8668,8 +8668,8 @@ def cli(self, vrf=None, address_family=None, summary=None, value=None, output=No
         # *>i192.168.111.0/24       10.189.99.98                                                    0       0 i
         # *> 10.7.7.7/32        10.10.10.107             0             0 65107.65107 ?
         p13 = re.compile(r'^(?P<status_codes>(i|s|x|S|d|h|\*|\>|\s)+)'
```

Comment:
Please comment above this what you're now matching

Reply:
add an example that requires this change on top of the pattern declaration

Reply:
@Kani999, kindly do not resolve the conversion unless you address the comments. You have to add the example for which you have made the changes in regex pattern. Kindly do so and then only resolve the conversion.

Also, if you have made changes to `ShowBgpVrf` parser class then you must add the unit tests like you added for `ShowBgpAddressFamily`. Kindly do so and then resolve the conversation.

Reply:
Ok, Difference between Old and New regex:

- Old one: https://regex101.com/r/w6F0eT/1
- New One: https://regex101.com/r/18lHd3/1